torch/csrc/jit/script/lexer.h

```diff
@@ -55,6 +55,7 @@ namespace script {
   _(TK_ELSE, "else", "else")                     \
   _(TK_ELIF, "elif", "elif")                     \
   _(TK_WHILE, "while", "while")                  \
+  _(TK_EXPR, "expression", "")                   \
```

Comment:
The name is a bit confusing because it is actually a statement. Maybe TK_EXPR_STATEMENT?